### PR TITLE
use paths instead of subdomains in takeout connected to #40

### PR DESCRIPTION
**Pull request by homeyer**

### CODE REVIEW

File: bin/build.sh [hunk 1]
```diff
@@ -48,8 +48,8 @@ echo -e "${grey}  Packaging waffle.io-migrations...${reset}"
 docker save --output="waffleio-takeout/images/waffle.io-migrations.tar" quay.io/waffleio/waffle.io-migrations
 echo -e "${grey}  Packaging waffle.io-rally-integration...${reset}"
 docker save --output="waffleio-takeout/images/waffle.io-rally-integration.tar" quay.io/waffleio/waffle.io-rally-integration
-echo -e "${grey}  Packaging nginx-proxy...${reset}"
-docker save --output="waffleio-takeout/images/nginx-proxy.tar" jwilder/nginx-proxy
+echo -e "${grey}  Packaging takeout-nginx...${reset}"
```
Comment (burtonjc) [inline bin/build.sh]: All our other images are called `waffle-*`, might be weird for admins to have to know both prefixes.

File: bin/install.sh [hunk 2]
```diff
@@ -38,13 +38,13 @@ echo HOST_NAME=$hostName >> $envFile
...
+echo WAFFLE_API_BASE_URL="https://${hostName}/api" >> $envFile
 sed -n '/WAFFLE_HOOKS_SERVICE_URI/!p' $envFile > tmp.list && mv tmp.list $envFile
-echo WAFFLE_HOOKS_SERVICE_URI="https://hooks.${hostName}" >> $envFile
+echo WAFFLE_HOOKS_SERVICE_URI="https://${hostName}/hooks" >> $envFile
 sed -n '/RALLY_INTEGRATION_BASE_URL/!p' $envFile > tmp.list && mv tmp.list $envFile
-echo RALLY_INTEGRATION_BASE_URL="https://rally.${hostName}" >> $envFile
+echo RALLY_INTEGRATION_BASE_URL="https://${hostName}/integrations/rally" >> $envFile
 sed -n '/POXA_HOST/!p' $envFile > tmp.list && mv tmp.list $envFile
-echo POXA_HOST="poxa.${hostName}" >> $envFile
+echo POXA_HOST="${hostName}" >> $envFile
```
Comment (burtonjc) [inline bin/install.sh]: so you cannot reach the poxa front end anymore?

File: bin/install.sh [hunk 3]
```diff
@@ -316,6 +316,13 @@ then
   echo WAFFLE_DB_ENCRYPTION_KEY=$encryptionkey >> $envFile
 fi
 
+# WAFFLE_DB_SIGNING_KEY
+if [ -z "$WAFFLE_DB_SIGNING_KEY" ];
+then
+  signingKey=$(openssl rand -base64 64 | tr -d '\n') # remove random newline in middle of key
+  echo WAFFLE_DB_SIGNING_KEY=$signingKey >> $envFile
+fi
```
Comment (burtonjc) [inline bin/install.sh]: Awesome! Thank you for doing this!

File: bin/install.sh [hunk 4]
```diff
@@ -401,9 +401,7 @@ docker load --input images/waffle.io-migrations.tar
 echo -ne '[##################         ] (66%)\r'
 docker load --input images/waffle.io-rally-integration.tar
 echo -ne '[#####################      ] (77%)\r'
-docker load --input images/waffle.io-admin.tar
-echo -ne '[########################   ] (88%)\r'
-docker load --input images/nginx-proxy.tar
+docker load --input images/takeout-nginx.tar
```
Comment (burtonjc) [inline bin/install.sh]: Yeah, I think this should be called `waffle-nginx` so the naming convention is consistent.

**Comment by burtonjc** [comment]
It would be nice if restarting an individual container also restarted nginx.

File: bin/build.sh (hunk 1 above)
Comment (homeyer) [inline bin/build.sh]: the image is still named `waffle-nginx`.

File: bin/install.sh (hunk 2 above)
Comment (homeyer) [inline bin/install.sh]: solution: let's forward requests to port `8000` to `poxa`, but keep `/app` and `/apps` how they are, so opening port 8000 to the wild is optional.

Comment (homeyer) [inline bin/install.sh]: nope, poxa doesn't like running on a path. Instead, we can manually restart poxa to expose the port if we need the console:

```
sudo service waffle-poxa stop
docker run -d --name waffle-poxa --env-file /etc/waffle/environment.list -p 8080:8080 quay.io/waffleio/poxa
sudo service waffle-nginx restart
```

**Pull request merged**